tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_Enthalpy.py

```diff
@@ -0,0 +1,156 @@
+        ]
+        cos_coefficients = [
+            -0.01080996024705052,
+            -0.003421193490191067,
+            0.012325774692378716,
+            0.004367136076912163,
+            -0.00020374276952538073,
+        ]
+        transition_delta_epsilon = 0.0
+
+        ## Low Density Equation:
+
+        # Generate python spectral equation of state.
+        # low_eos_pyth = spectral_python.spectral_eos(
+        # reference_density=low_eos_reference_density,
+        # reference_pressure=low_eos_reference_pressure,
+        # coefficients=low_eos_spectral_coefficients,
+        # upper_density=low_eos_upper_density,
+        # )
+        # Generate spectre spectral equation of state.
+        low_eos_spectre = EoS.Spectral(
+            reference_density=low_eos_reference_density,
+            reference_pressure=low_eos_reference_pressure,
+            spectral_coefficients=low_eos_spectral_coefficients,
+            upper_density=low_eos_upper_density,
+        )
+
+        ## Enthalpy Equations:
+
+        # Python Implementation (interpolator + evaluator):
+        # enthalpy_interpolator_pyth = enthalpy_python.enthalpy_interpolator(
+        # poly_coeffs=polynomial_coefficients,
+        # sin_coeffs=sin_coefficients,
+        # cos_coeffs=cos_coefficients,
+        # max_density=max_density,
+        # min_density=min_density,
+        # rho_scale=reference_density,
+        # trig_scale=trig_scale,
+        # )
+
+        # eos_pyth = enthalpy_eos(
+        # eos=enthalpy_interpolator_pyth,
+        # min_energy_density=(
+        # enthalpy_eos.min_energy_density_of_low_density_eos(
+        # low_eos_pyth, enthalpy_interpolator_pyth.min_density
+        # )
+        # ),
+        # low_density_eos=low_eos_pyth,
+        # )
+
+        ## Generate "Spectre" Equation of state
+        eos_spectre = EoS.EnthalpySpectral(
+            reference_density=reference_density,
+            max_density=max_density,
+            min_density=min_density,
+            trig_scale=trig_scale,
+            polynomial_coefficients=polynomial_coefficients,
+            sin_coefficients=sin_coefficients,
+            cos_coefficients=cos_coefficients,
+            low_density_eos=low_eos_spectre,
+            transition_delta_epsilon=transition_delta_epsilon,
+        )
+        self.assertTrue(
+            isinstance(eos_spectre, EoS.RelativisticEquationOfState1D)
+        )
+        # Only testing p(rho) because the class is tested thoroughly in C++
+        density = np.geomspace(4.6e-4, 5e-3, 10)
+        pressure = eos_spectre.pressure_from_density(
+            Scalar[DataVector](density)
+        )
+        """
+        These are the pressures calculated in python implementation
+        testing the bindings. The python implementation will be added
+        in a second merge request.
+        """
+        pressures_pyth = [
+            0.00001530284908801628381003201440080374595709145069122314453125,
```

Comment:
How did you get this many digits? :D And why do they all end with 5? :D Just limit this to at most 16 digits (double precision)